apps/betterangels-backend/shelters/models.py

```diff
@@ -273,16 +273,19 @@ def __str__(self) -> str:
 class InteriorPhoto(models.Model):
     file = AsyncFileField(upload_to=upload_path)
     shelter = models.ForeignKey(Shelter, on_delete=models.CASCADE, related_name="interior_photos")
+    created_at = models.DateTimeField(auto_now_add=True)
```

Comment:
Maybe we can use basemodel here?

Reply:
lol. I was gonna and then didn't because we don't need `updated_at`. will change